docs/introduction/Car-registry-tutorial.rst

```diff
@@ -500,7 +492,7 @@ As a first parameter we pass reference to SidechainNodeView, second reference is
                 this.fee = fee;
             }
         }
-    * Specify transaction itself. Because we recall our sell order then isSeller parameter during transaction creation is set to false.
+    * Specify transaction itself. Because we recalled our sell order, the isSeller parameter during transaction creation is set to false.
```

Comment:
@zennermac 

    * Specify the transaction itself. Because we recalled our sell order, the isSeller parameter during transaction creation is set to false.